docs/book/src/user/quick-start.md

```diff
@@ -417,7 +417,7 @@ clusterctl init --infrastructure aws
 
 Download the latest release; on Windows, type:
 ```
-curl.exe -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api" gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"clusterawsadm-windows-amd64" version:">=2.0.0"}} -o clusterawsadm.exe
+curl.exe -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api" gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"clusterawsadm-windows-amd64.exe" version:">=2.0.0"}} -o clusterawsadm.exe
```

Comment:
```suggestion
curl.exe -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api-provider-aws" gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"clusterawsadm-windows-amd64.exe" version:">=2.0.0"}} -o clusterawsadm.exe
```

I think this needs to be the capa repo, also for the other OS's for clusterawsadm.

The link generated is https://github.com/kubernetes-sigs/cluster-api/releases/download/v2.3.0/clusterawsadm-windows-amd64.exe and still does not work for me.

Reply:
Yes!

Reply:
Sorry guys. Completely missed this PR. Any task pending from my side?

Reply:
Yes https://github.com/kubernetes-sigs/cluster-api/pull/9870/files#r1424943989

The binary is hosted in the release artifacts of the cluster-api-provider-was repo so we can't use cluster-api as the repo